lib/vagrant-openshift/version.rb

```diff
@@ -16,6 +16,6 @@
 
 module Vagrant
   module Openshift
-    VERSION = "2.0.6"
+    VERSION = "2.0.7"
```

Comment:
This is ok.  But just as an FYI you don't need to add the version bump with your commits.